src/applications/personalization/profile-2/components/MobileMenuTrigger.jsx

```diff
@@ -102,7 +102,7 @@ const MobileMenuTrigger = ({
   const buttonClasses = classnames({ fixed: isMenuTriggerPinned });
 
   return (
-    <div className="va-btn-sidebarnav-trigger">
+    <div className="va-btn-mobile-nav-trigger">
```

Comment:
Changing the class name for this React version of the sidenav. When we were using `va-btn-sidebarnav-trigger` some unwanted site-wide CSS was cascading down into this component.